src/Castle.Core/Castle.Core-VS2017.csproj

```diff
@@ -10,6 +10,14 @@
     <AssemblyOriginatorKeyFile>..\..\buildscripts\CastleKey.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>True</SignAssembly>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <None Remove="DynamicProxy\DynProxy.snk" />
```

Comment:
The remove is totally unnecessary. I am having a grand old time trying to keep this tidy. VS create new item groups of it's own random choosing. Some of the rules around how it maintains these files are a little `rough round the edges`.